application/src/main/java/org/togetherjava/tjbot/commands/moderation/attachment/BlacklistedAttachmentListener.java

```diff
@@ -91,7 +93,7 @@ private boolean doesMessageContainBlacklistedContent(Message message) {
         List<Message.Attachment> attachments = message.getAttachments();
         return attachments.stream()
             .anyMatch(attachment -> blacklistedFileExtensions
-                .contains(attachment.getFileExtension()));
+                .contains(Objects.requireNonNull(attachment.getFileExtension()).toLowerCase()));
```

Comment:
`toLowerCase()` is platform dependent (for example `i` ends up as `ı`  (dotless i) on a turkish machine). Replace with `toLowerCase(Locale.US)`, then its correct.